elasticdl/python/master/main.py

```diff
@@ -227,31 +231,14 @@ def main():
         worker_args = [
             "-m",
             "elasticdl.python.worker.main",
-            "--model_zoo",
-            args.model_zoo,
             "--master_addr",
             master_addr,
-            "--log_level",
-            args.log_level,
-            "--dataset_fn",
-            args.dataset_fn,
-            "--loss",
-            args.loss,
-            "--optimizer",
-            args.optimizer,
-            "--eval_metrics_fn",
-            args.eval_metrics_fn,
-            "--model_def",
-            args.model_def,
             "--job_type",
```

Comment:
Why these args are kept here?

Reply:
They are not the arguments that parsed from command line.

Reply:
Got it.